### PR TITLE
mention prereqs and flash instructions in bsp readmes

### PR DESCRIPTION
**Pull request by jacobrosenthal**
Related to https://github.com/atsamd-rs/atsamd/issues/18

Are we interested in having more resources in our bsp readmes to guide new users to getting started?

Happy to bikeshed any of this, and or add to other bsps before merge

### CODE REVIEW

**Comment by sajattack** [comment]
I think a lot of this can go in the main readme to avoid duplication.

**Comment by sajattack** [comment]
Actually, I take that back, it only applies to the adafruit boards, and it's good to have the instructions on the crates.io for each board.

**Comment by jacobrosenthal** [comment]
Ill pr all the adafruit boards. Anyone else who managed those bsps please feel free to comment on general language

**Comment by jacobrosenthal** [comment]
I think were already holding on this, but lets hold on this until we confirm a path for linux users. 
https://github.com/jacobrosenthal/hf2-rs/issues/9

As well as windows users
https://github.com/jacobrosenthal/hf2-rs/issues/7

File: boards/trinket_m0/README.md [hunk 1]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit Trinket M0
 board](https://www.adafruit.com/product/3500).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/trinket_m0/README.md]: Trinket M0 is `thumbv6m`

File: boards/metro_m0/README.md [hunk 2]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit Metro M0
 board](https://www.adafruit.com/product/3505).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/metro_m0/README.md]: Metro M0 is `thumbv6m`

File: boards/itsybitsy_m0/README.md [hunk 3]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit ItsyBitsy M0
 board](https://www.adafruit.com/product/3727).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/itsybitsy_m0/README.md]: ItsyBitsy M0 is `thumbb6m`

File: boards/gemma_m0/README.md [hunk 4]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit Gemma M0
 board](https://www.adafruit.com/product/3501).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/gemma_m0/README.md]: Gemma M0 is `thumbb6m`

File: boards/feather_m0/README.md [hunk 5]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit Feather M0
 board](https://www.adafruit.com/product/2772).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/feather_m0/README.md]: Feather M0 is `thumbb6m`

File: boards/circuit_playground_express/README.md [hunk 6]
```diff
@@ -3,8 +3,24 @@
 This crate provides a type-safe API for working with the [Adafruit Circuit Playground Express
 board](https://www.adafruit.com/product/3333).
 
-## Examples?
+## Prerequisites
+* Install the cross compile toolchain `rustup target add thumbv7em-none-eabihf`
```
Comment (jessebraham) [inline boards/circuit_playground_express/README.md]: Circuit Playground Express is `thumbb6m`

**Comment by jacobrosenthal** [comment]
Addressed. Thanks!

**Pull request merged**